assets/img/broom.svg

```diff
@@ -0,0 +1,67 @@
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
+<g>
+	<g>
+		<path d="M423.256,104.181c-3.763-3.067-9.301-2.5-12.367,1.263l-4.272,5.244c-3.066,3.763-2.5,9.301,1.263,12.367
+			c1.631,1.329,3.595,1.975,5.546,1.975c2.552,0,5.083-1.105,6.82-3.238l4.272-5.244
+			C427.586,112.783,427.02,107.247,423.256,104.181z"/>
+	</g>
+</g>
+<g>
+	<g>
+		<path d="M484.235,28.513c-0.861-8.438-4.959-16.037-11.536-21.393c-13.574-11.058-33.618-9.009-44.675,4.564L271.802,203.367
+			l-0.13-0.105c-6.771-5.517-15.8-7.291-24.154-4.749c-8.354,2.541-14.866,9.043-17.418,17.394l-6.5,21.26l-2.071-1.688
+			c-1.807-1.472-4.125-2.172-6.445-1.929c-2.319,0.237-4.449,1.385-5.922,3.193l-25.591,31.416
+			c-1.472,1.807-2.167,4.125-1.929,6.445c0.237,2.319,1.385,4.449,3.193,5.921l0.431,0.352l-54.973,21.047
+			c-4.533,1.735-6.802,6.818-5.067,11.352c1.341,3.5,4.674,5.649,8.211,5.649c1.045,0,2.107-0.188,3.141-0.584l64.407-24.659
+			l48.542,39.542l-27.732,153.76l-80.401-65.494l34.335-42.15c3.066-3.765,2.5-9.302-1.263-12.367
+			c-3.763-3.066-9.302-2.501-12.367,1.262l-34.336,42.15l-13.968-11.378l18.381-22.564c3.066-3.765,2.5-9.302-1.263-12.367
+			c-3.765-3.066-9.301-2.501-12.367,1.262l-18.381,22.565l-46.448-37.837l36.168-13.847c4.533-1.735,6.802-6.818,5.067-11.352
+			c-1.736-4.533-6.819-6.8-11.352-5.065l-50.354,19.278c-2.93,1.122-5.04,3.72-5.537,6.816c-0.497,3.099,0.695,6.226,3.127,8.206
+			l191.414,155.922c1.593,1.297,3.559,1.976,5.553,1.976c1.025,0,2.058-0.179,3.05-0.546c2.918-1.081,5.048-3.623,5.6-6.684
+			l28.721-159.245c1.133,0.502,2.333,0.779,3.538,0.779c2.551,0,5.083-1.105,6.82-3.238l25.591-31.415
+			c1.472-1.808,2.167-4.127,1.929-6.446c-0.193-1.889-1-3.647-2.267-5.033l19.394-9.494c8.058-3.945,13.478-11.62,14.499-20.534
+			c1.021-8.913-2.521-17.615-9.476-23.282l-4.155-3.385l58.888-72.199c3.068-3.761,2.506-9.299-1.256-12.367
+			c-3.761-3.07-9.3-2.508-12.367,1.256l-58.895,72.208l-21.983-17.908L441.654,22.788c2.392-2.936,5.783-4.764,9.55-5.15
+			c3.766-0.382,7.459,0.721,10.395,3.112c2.935,2.392,4.764,5.783,5.149,9.549c0.386,3.767-0.72,7.458-3.112,10.393l-36.444,44.739
+			c-3.066,3.763-2.5,9.301,1.263,12.367c3.762,3.065,9.301,2.5,12.367-1.263l36.444-44.739
+			C482.622,45.22,485.098,36.952,484.235,28.513z M267.452,325.147l-2.847-2.319c-0.007-0.006-0.014-0.012-0.021-0.016
+			l-61.832-50.367l14.487-17.785l5.287,4.307c0.015,0.013,0.032,0.025,0.047,0.037l56.513,46.037
+			c0.015,0.012,0.03,0.025,0.046,0.036l2.807,2.287L267.452,325.147z M314.098,260.495c2.319,1.888,3.454,4.676,3.114,7.647
+			c-0.341,2.972-2.077,5.431-4.763,6.746l-26.572,13.007l-47.559-38.742l8.594-28.109c0.851-2.785,2.937-4.867,5.722-5.715
+			c2.787-0.846,5.678-0.279,7.936,1.561L314.098,260.495z"/>
+	</g>
+</g>
+<g>
```

Comment:
@ashishdommety a lot of empty groups - might be worthwhile running your SVGs through SVGO https://github.com/svg/svgo